src/UI/BodyPanel.java

```diff
@@ -0,0 +1,233 @@
+
+            },
+            new String [] {
+                "Name", "Type", "Size", "Date"
+            }
+        ) {
+            Class[] types = new Class [] {
+                java.lang.String.class, java.lang.String.class, java.lang.String.class, java.lang.String.class
+            };
+            boolean[] canEdit = new boolean [] {
+                false, false, false, false
+            };
+
+            public Class getColumnClass(int columnIndex) {
+                return types [columnIndex];
+            }
+
+            public boolean isCellEditable(int rowIndex, int columnIndex) {
+                return canEdit [columnIndex];
+            }
+        });
+        tableRigth.addFocusListener(new java.awt.event.FocusAdapter() {
+            public void focusGained(java.awt.event.FocusEvent evt) {
+                tableRigthFocusGained(evt);
+            }
+        });
+        jScrollPaneRigth.setViewportView(tableRigth);
+
+        jSplitPane.setRightComponent(jScrollPaneRigth);
+
+        this.add(jSplitPane, java.awt.BorderLayout.CENTER);
+    }
+    
+    /**
+     * Method to add the roots to panel
+     * @param evt 
+     */
+    public void formWindowOpened(java.awt.event.WindowEvent evt) {                                  
+        File[] drives = File.listRoots();
+        for (int index = 0; index < drives.length; index++) {
+            checkBoxSelectDrive.addItem(drives[index].getPath());
+        }
+    }                                 
+
+    /**
+     * Method to set the file path 
+     * @param evt 
+     */
+    private void checkBoxSelectDriveActionPerformed(java.awt.event.ActionEvent evt) {                                                    
+        filePath.setText(checkBoxSelectDrive.getSelectedItem().toString());
+    }                                                   
+
+    /**
+     * Method to list the file when the user selects Go button
+     * @param evt 
+     */
+    private void jButtonGoActionPerformed(java.awt.event.ActionEvent evt) {                                          
+        listFiles(filePath.getText());
+    }    
+    
+    /**
+     * Method to list all files of a given path 
+     * @param pathFile where the items will be retrieved 
+     */
+    public void listFiles(String pathFile) {
+        Path path = Paths.get(pathFile);
+        File f = path.toFile();
+        File[] files = f.listFiles();
+        
+        DefaultTableModel model = null;
+        
+        if (onLeft) {
+            model = (DefaultTableModel) tableLeft.getModel();
+        } else {
+            model = (DefaultTableModel) tableRigth.getModel();
+        }
+        
+        model.getDataVector().removeAllElements();
+        
+        for (int index = 0; index < files.length; index++) {
+            Date d = new Date(files[index].lastModified());
+            String[] row= {files[index].getName(), files[index].isFile() ? "FILE" : "DIR", String.valueOf(files[index].length()/1024), d.toGMTString()};
```

Comment:
more than 100 characters in a line and you need spaces before and after an assignation